table/block_based/full_filter_block.cc

```diff
@@ -168,16 +169,8 @@ bool FullFilterBlockReader::MayMatch(
 
   assert(filter_block.GetValue());
 
-  if (filter_block.GetValue()->data.size() != 0) {
-    assert(table());
-    assert(table()->get_rep());
-
-    std::unique_ptr<FilterBitsReader> filter_bits_reader(
-        table()->get_rep()->filter_policy->GetFilterBitsReader(
-            filter_block.GetValue()->data));
-    assert(filter_bits_reader != nullptr);
-
-    if (filter_bits_reader->MayMatch(entry)) {
+  if (filter_block.GetValue()->filter_bits_reader()) {
```

Comment:
Consider stuffing `filter_block.GetValue()->filter_bits_reader()` in a local variable